cat-launcher/src-tauri/src/fetch_releases/repository/sqlite_releases_repository.rs

```diff
@@ -21,6 +21,34 @@ impl SqliteReleasesRepository {
   pub fn new(pool: Pool) -> Self {
     Self { pool }
   }
+
+  pub fn get_release_notes(
```

Comment:
Release notes should not be part of the existing releases repository. Release notes should have their own table, and their own repository.